index.html

```diff
@@ -243,7 +253,7 @@ <h2 class="one">Skills</h2>
             </ul>
         </section>
 
-        <section class="footer-nav">
+        <section class="footer-nav" id="contact">
             <div class="footer-grid">
                 <h3 class="paragraph">
                     I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
```

Comment:
## [OPTIONAL]
- [ ]  By following HTML semantic  tags kindly use the `<p>` tag  for this text as it is considered as a paragraph and not as a heading `<h3>`, also remove make sure it is well centered

![image](https://user-images.githubusercontent.com/65068771/150795492-fd20befb-a21d-40d9-9b7f-02e4d217a59b.png)